.jenkins/caffe2/test.sh

```diff
@@ -136,15 +136,15 @@ pip install --user pytest-sugar
 # torchvision tests #
 #####################
 if [[ "$BUILD_ENVIRONMENT" == *onnx* ]]; then
-  pip install -q --user git+https://github.com/pytorch/vision.git
+  pip install -q --user torchvision=0.5.0
```

Comment:
Seems 0.5.0 is not available in pip repository yet.

Try pip install -q --user git+https://github.com/pytorch/vision.git@v0.5.0?

Reply:
Yes, use the git hash instead of relying on the package from pip. Plus, we would probably have issues with binary compatibility, so it's better to compile torchvision

Reply:
Thanks Lu! I've updated it.